src/app/modules/shared/components/details-fields/details-fields.component.spec.ts

```diff
@@ -468,6 +468,31 @@ describe('DetailsFieldsComponent', () => {
     expect(endDateInput.value).not.toEqual(startDateInput.value);
     expect(startDateInput.value).toEqual(expectedStartDate);
   });
+
+  it('on get actual date should return the current date', () => {
+    const actualDateExpected = new Date().toISOString().split('T')[0];
+
+    expect(component.getCurrentDate()).toEqual(actualDateExpected);
+  });
+
+  it('on the input with id #start_date we could get the id and max value', () => {
+   fixture.detectChanges();
+   const expectDate = new Date().toISOString().split('T')[0];
+   const slectInput: HTMLInputElement = fixture.debugElement.
```

Comment:
I think if you want to retrieve the selecInput. You only need to queryselector by the id.

Reply:
Also, the name `slectInput` should be `startDateInput` or `dateInput`